raiden/src/raiden.ts

```diff
@@ -168,7 +159,19 @@ export class Raiden {
                         )
                       : Zero,
                   ),
-                }),
+                  capacity = channel.own.deposit.add(balance);
```

Comment:
Quite difficult to read with two ternary operators right above. Could be prevented via eslint's `one-var: ["always"]`

Reply:
Please, no... `one-var` is horrendous, it'd make our code a spam of pointless `const`s and `let`s, while a properly configured IDE should highlight the assignment operations.
I know the multiple ternary operators are a little bit hard to follow sometimes, but that's due to TypeScript's hard requirements on type safety (as these properties aren't always available).. This will get improved on TS 3.7, which should ship with [optional chaining](https://github.com/tc39/proposal-optional-chaining) and [nullish coalescing](https://github.com/tc39/proposal-nullish-coalescing) operators. We may improve that here as well by making the `balanceProof` and `locks` properties set to defaults earlier, so the type narrowing on `state` can infere their presence.